src/main/java/com/microsoft/azure/functions/worker/reflect/DefaultClassLoaderProvider.java

```diff
@@ -30,18 +30,23 @@ public ClassLoader createClassLoader() {
     return classLoader;
   }
 
-  @Override
-  public void addUrl(URL url) throws IOException {
+  public void addUrl(URL url, String source) throws IOException {
     if (urls.contains(url)) {
       return;
     }
-    
-    WorkerLogManager.getSystemLogger().info("Loading file URL: " + url);    
-
+    WorkerLogManager.getSystemLogger().info("Loading " + source + " file URL: " + url);
     urls.add(url);
+    addUrlToSystemClassLoader(url);
+  }
 
+  @Override
+  public void addCustomerUrl(URL url) throws IOException {
+    addUrl(url, "customer");
```

Comment:
```suggestion
    addUrl(url);
```
We can review if source is needed